doc/api/buffer.md

```diff
@@ -788,6 +788,28 @@ const buf = Buffer.from([0x62, 0x75, 0x66, 0x66, 0x65, 0x72]);
 
 A `TypeError` will be thrown if `array` is not an `Array`.
 
+`Buffer.from(array)` may also use the internal `Buffer` pool like
+[`Buffer.allocUnsafe()`] does.
+
+```js
+const nodeBuffers = Buffer.from([0, 1, 2, 3, 4, 5, 6, 7, 8, 9]);
+
+console.log(nodeBuffers.offset);
+// prints: 344
```

Comment:
This is not always correct. It may return 0, and any subsequent call will probably increase it.
So it doesn't always print 344.